src/app/auth/rules/RulesModal.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Something is wrong with formatting in this file, isn't it?

Reply:
I don't see)
What?

Reply:
Semicolon
Prettier auto inserts them

Reply:
Today work from home and prettier wasn't configure.